wasm/src/project_system.rs

```diff
@@ -187,14 +188,41 @@ pub(crate) fn get_manifest_transformer(js_val: JsValue, _: String) -> Option<Man
 
     };
 
+    let lints: Vec<LintConfig> = match js_sys::Reflect::get(&js_val, &JsValue::from_str("lints")) {
+        Ok(v) => match v.dyn_into::<js_sys::Array>() {
+            Ok(arr) => arr
+            .into_iter()
+            .filter_map(|x| {
+                    let lint = js_sys::Reflect::get(&x, &JsValue::from_str("lint"))
+                        .expect("expected lint according to typescript bindings");
+                    let level = js_sys::Reflect::get(&x, &JsValue::from_str("level"))
+                        .expect("expected lint according to typescript bindings");
+                    let lint = lint.as_string().expect("manifest callback returned {lint:?}, but we expected a string representing a lint name");
+                    let level = level.as_string().expect("manifest callback returned {level:?}, but we expected a string representing a lint level");
+
+                    let kind = format!(r#""{lint}""#);
+                    let level = format!(r#""{level}""#);
+                    let kind: LintKind = serde_json::from_str(&kind).ok()?;
+                    let level: LintLevel = serde_json::from_str(&level).ok()?;
+
+                    Some(LintConfig { kind, level })
+                })
```

Comment:
OK, since you already declared `LintConfig` as `Serializable`, this works quite nicely. I tested this:

```suggestion
                .filter_map(|x| serde_wasm_bindgen::from_value::<LintConfig>(x).ok())
```

I'm pretty sure the same could be applied to this entire function (since all of the `Manifest` struct is serializable) but maybe it's best not to risk it right now.

Reply:
Yes, this indeed works!